README.md

```diff
@@ -81,10 +81,44 @@ Finally, we look at the procedural steps adopted by the framework:
+   2. Docker: Docker image is built - for this docker file needs to be provided by the user.
+5. Serverless.com framework deploys the service defined in the service.yml. (`serverless deploy`)
+6. Serverless.com return a list of endpoints and routes for every function defined.
+7. Benchmarking is performed.
+8. Once all the experiment are finished, the service is removed using the serverless.com framework. (`serverless remove`)
+
+### Serverless.com & Cloud Provider Capabilities
+While serverless.com framework is a powerful tool when it comes to deploying Lambda Functions to AWS, its capabilities are more limited with other providers. The following table shows serverless.com features vs. different providers whose deployment was considered to be automated.
+
+|  |AWS Lambda | Azure Function | Google Cloud Run | Knative | Google Cloud Function *                  | Alibaba                                  |
+|--|----|-------------|------------------|---------|------------------------------------------|------------------------------------------|
+|Deploy function - zip| Yes | Yes | No               | N/A     | Yes                                      | Yes                                      |
+|Deploy function - docker|Yes|No (documentation does not mention it)**| No               | Yes     | No (documentation does not mention it)** | No (documentation does not mention it)** |
+|Python Runtime|Yes|Yes| No               | Yes     | Yes                                      | ??**                                     |
+|Go Runtime|Yes|No| No               | Yes     | Yes                                      | ??**                                     |
+|Java Runtime|Yes|No| No               | ??**    | No                                       | ??**                                     |
+|Node JS|Yes|Yes| No               | Yes     | Yes                                      | Yes                                      |
+|HTTP trigger|Yes|Yes| No               | Yes     | Yes                                      | Yes                                      |
+|Resource Management (e.g. buckets databases)|Yes|Yes (at least it looks like it)| No           | ??**    | ??**                                     | ??**                                     ||No**|No**|
+*experimental version, not meant for production yet
+
+**needs to be empirically verified
+
 ## Data Transfer Measurement
 We integrate all necessary server-side functionality into a single function that we call a _measurement function_. This approach is similar to that taken in [40] and other serverless performance evaluation frameworks. A measurement function can perform up to three tasks, depending on the use case:
```

Comment:
Have we referenced this -> [40]?

Reply:
I don not know what it is supposed to refer to, maybe the citation from the IISWC paper? 
![image](https://github.com/vhive-serverless/STeLLAR/assets/44442465/1a43c208-2c93-4594-b697-a2189ec1401a)